doc/source/building/build_docker_container.rst

```diff
@@ -1,190 +1,91 @@
-
-.. code:: bash
-
-   $ zypper in umoci skopeo
-
-Building the Image
-~~~~~~~~~~~~~~~~~~
-
-Run the following KIWI command in order to build the base image example.
-
-.. code:: bash
-
-    $ sudo kiwi-ng --type docker system build \
-        --description kiwi-descriptions/suse/x86_64/suse-leap-42.2-docker \
-        --target-dir /tmp/myimage
-
-Find the image with the suffix :file:`.docker.tar.xz` below
-:file:`/tmp/myimage`. For testing the docker image it's required
-to have the docker subsystem installed and running. The following
-command can be used to check if docker is installed and functional
-on your system:
-
-.. code:: bash
-
-    $ docker images
-
-The command above lists all available docker images. If the docker
-daemon is running and responsive a list output is expected which
-could also be empty if no images are available. In any other case
-an error message is displayed which needs to be resolved prior to
-continue with the next steps.
-
-Once docker works on your system the KIWI built image needs to be
-loaded into the docker system with the following command:
-
-.. code:: bash
-
-    $ docker load -i Docker-Leap-42.2.x86_64-1.0.0.docker.tar.xz
-
-On success the docker image is available as part of the images list as follows:
-
-.. code:: bash
-
-    $ docker images
-
-    REPOSITORY          TAG  ...
-    opensuse            42.2 ...
-
-In order to run an interactive shell session of the loaded container
-the following command needs to be called:
-
-.. code:: bash
-
-    $ docker run -it opensuse:42.2 /bin/bash
-
-Container Configuration
-~~~~~~~~~~~~~~~~~~~~~~~
-
-A docker container includes a set of information also called metadata.
-The metadata information is provided within the KIWI XML description
-as part of the ``<containerconfig>`` section. The following parameters
-can be specified:
+The Docker configuration metadata is provided to KIWI as part of the
+:ref:`XML description file <decription_components>` using the
+``<containerconfig>`` section. The following parameters
+can be specified (more detailed schema documentation
+:ref:`here <k.image.preferences.type.containerconfig>`):
 
+* ``name``: Specifies the repository name of the Docker image.
+* ``tag``: Sets the tag of the Docker image.
 * ``maintainer``: Specifies the author field of the container.
-* ``cmd``: Provides the default execution parameters of the container.
-* ``label``: Adds custom metadata to an image, is key-value pair.
+* ``subcommand``: Provides the default execution parameters of the container.
+  Equivalent of the `CMD` directive of a Docker file.
+* ``labels``: Adds custom metadata to an image, is key-value pair. Equivalent
+  of the `LABEL` directive of a Docker file.
 * ``expose``: Informs at which ports is the container listening at runtime.
-* ``env``: Sets an environment value, is key-value pair.
+  Equivalent of the `EXPOSE` directive of a Docker file.
+* ``environment``: Sets an environment values using key-value pairs.
```

Comment:
Plural or singular? I assume plural here, in that case I would rephrase to:

    Set environment values using key-value pairs.

It would be helpful to add *how* a single key-value pair can be separated from another one. Use spaces, commas, or something different?